workflows/sarek.nf

```diff
@@ -335,10 +335,13 @@ workflow SAREK {
     bwa                    = params.bwa                     ? Channel.fromPath(params.bwa).collect()       : PREPARE_GENOME.out.bwa
     bwamem2                = params.bwamem2                 ? Channel.fromPath(params.bwamem2).collect()   : PREPARE_GENOME.out.bwamem2
     dragmap                = params.dragmap                 ? Channel.fromPath(params.dragmap).collect()   : PREPARE_GENOME.out.hashtable
+    sentieon               = params.sentieon                ? Channel.fromPath(params.sentieon).collect()  : PREPARE_GENOME.out.sentieon
+
     // Gather index for mapping given the chosen aligner
     index_alignement = params.aligner == "bwa-mem" ? bwa :
```

Comment:
```suggestion
    index_alignement = params.aligner == "bwa-mem" || (params.sentieon && params.aligner == "sentieon-bwamem") ? bwa :
```

Reply:
```suggestion
    index_alignement = (params.aligner == "bwa-mem" || params.aligner == "sentieon") ? bwa :
        params.aligner == "bwa-mem2" ? bwamem2 :
        dragmap
```
and then I delete line 338 (the one referring to `params.sentieon` )